stable/phabricator/templates/ingress.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.ingress.enabled -}}
+{{- $serviceName := include "phabricator.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    app: {{ template "phabricator.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "phabricator.fullname" . }}
+  annotations:
+    {{- range $key, $value := .Values.ingress.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  rules:
+    {{- range $host := .Values.ingress.hosts }}
+    - host: {{ $host }}
+      http:
+        paths:
+          - path: {{ $ingressPath }}
+            backend:
+              serviceName: {{ $serviceName }}
+              servicePort: 80
+    {{- end -}}
+  {{- if .Values.ingress.tls }}
```

Comment:
I see that you are iterating over different hosts but all of them would use the same certificate, right?
Also, when would you need different hosts to configure the Phabricator ingress?

Reply:
When migrating from an old or hosted version. I'd want to support the old host name of `phabricator.example.com` but develop against `phabricator.k8.example.com` so that it just works when i update the dns entry. Kube lego can take some time to update certificates once its already generated. Its also the way several other charts are handling TLS.

Reply:
ok! Thanks for the exaplanation